### DISCUSSION TITLE
Guide on discussing impact of open source contributions.

### DISCUSSION THREAD

**Discussion by CBID2** [EddieHubCommunity/support#7530] [Ideas]
In the open source community, I haven't seen discussions on how contributors can mention their open source experience in their resume or during a job interview. Since there’s [a conversation about adding a Linkfree feature where people can highlight their open source contributions](https://github.com/EddieHubCommunity/LinkFree/issues/7164), I think EddieHub members(and open source contributors in general) would benefit greatly from a guide that provides tips on this topic. What do you think?

**Comment by eddiejaoude** [comment]
Thanks Chrissy, the issue with label discussion was raised first, so I will close this as duplicate https://github.com/EddieHubCommunity/LinkFree/issues/7164
